src/app/components/drawer/Projects/index.js

```diff
@@ -91,6 +91,13 @@ const Projects = () => {
                 }
               }
             }
+            trash_count
+            spam_count
+            search {
+              id
+              number_of_results
+            }
+            verification_statuses,
```

Comment:
Considering we're not doing the "add item" refactor in this PR, are these lines necessary now?

Reply:
Thanks for catching that :)